ship-it-inc/ship-it-fend
# #168086743 Set up redux

#### What does this PR do?
It sets up redux
#### Description of Task to be completed?
- set up redux

#### How should this be manually tested?
- Fetch the branch `ch-setup-redux-168086743`
- Check out into the branch `ch-setup-redux-168086743`
- run npm run dev
- You should be directed to the login ui page.

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/168086743](#168086743)
#### Screenshots (if appropriate)
NIl
